src/worker.rs

```diff
@@ -79,7 +80,20 @@ fn list_files(logger: &Logger, name: &str, recursive: bool) -> Result<Vec<File>,
             let mut in_file_entry = false;
             let (mut in_name, mut in_author, mut in_date, mut in_size) = (false, false, false, false);
             let (mut name, mut author, mut date, mut size, mut revision) = (None, None, None, None, None);
-
+            let sha_output = Command::new("svn")
+                .arg("propget")
+                .arg("git-commit")
+                .args("--revprop")
+                .args("-r")
+                .args("HEAD")
```

Comment:
This shouldn't be `HEAD`. It should be each unique revision. I think you'll have to build up the struct without these and then insert them into it since we don't want to make spurious requests.

Reply:
k

Reply:
Couldn't you insert a number in the format `HEAD~{}`?

Reply:
No. There are potentially multiple unique revision numbers.